components/ShopBuilder/Layouts/BasicLayout.js

```diff
@@ -1,43 +1,81 @@
 export const BasicLayoutContent = [
   {
-    content: "banner",
+    style: {},
+    content: {
+      message: 'i"m a banner i"m a banner i"m a banner'
+    },
+    contentType: "banner",
     id: `banner-${Date.now().toPrecision()}`
   },
   {
-    content: "store-name",
+    style: {},
+    content: {
+      title: "Store Header"
+    },
+    contentType: "store-name",
```

Comment:
So far we can bring in the components, we do need to in the future make the store name editable.